shared/js/initApp.js

```diff
@@ -240,6 +283,9 @@ if (appOptions.droplet) {
         return deferred;
       });
     }
+  } else if (appOptions.level.projectTemplateLevelName) {
+    // this is an embedded project
+    promise.then(dashboard.loadEmbeddedProject(appOptions.level.projectTemplateLevelName));
```

Comment:
That was the intention... if we can't load/initialize the project we should not load the level. Because, without the projects, the stage won't "work" because progress won't be saved across levels.

I could see both ways though, do you think we should still attempt to load the rest of the level?